integration/issue10244_container_exec_test.go

```diff
@@ -0,0 +1,132 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package integration
+
+import (
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/containerd/containerd/v2/integration/images"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIssue10244LoopbackTrue(t *testing.T) {
+	t.Log("Get containerd config")
+	configPath := "/etc/containerd/config.toml"
+	config, err := os.ReadFile(configPath)
+	if err != nil {
+		t.Skipf("Failed to read containerd config at path %s: %v", configPath, err)
```

Comment:
if the config is not present.. containerd will use the defaults.. so instead of skipping presume default values and continue.. that said it might be easier to just use an instance here with a manual config..

See: https://github.com/containerd/containerd/blob/d257459d50804ed090b02455cd6acc968a17e401/integration/container_volume_linux_test.go#L32